package.json

```diff
@@ -18,6 +18,7 @@
   },
   "precommit": [
     "lint",
+    "type-check",
     "test"
```

Comment:
It's a actually a node package called [pre-commit](https://www.npmjs.com/package/pre-commit), but yeah comes in handy